charts/pvault-server/templates/configmap-db-ca.yaml

```diff
@@ -3,9 +3,9 @@ kind: ConfigMap
 apiVersion: v1
 metadata:
   name: {{ include "pvault-server.fullname" . }}-db-ca-cert
-  labels:
-    {{- include "pvault-server.labels" . | nindent 4 }}
+  namespace: {{ include "pvault-server.namespace" . | quote }}
+  labels: {{- include "pvault-server.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
```

Comment:
What's customLabels? Should it be commonLabels as is the values.yaml file?

Reply:
`customLabels` is the key used by the `pvault-server.labels.standard` helper function.
`.Values.commonLabels` is still used as the content/value.